experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -759,7 +759,15 @@ export function revisionMetadataSourceFromInfo(
 ): RevisionMetadataSource {
 	const getIndex = (revision: RevisionTag): number | undefined => {
```

Comment:
This new implementation seems like a stretch of the original contract for this interface.
Either make a separate function or add a new parameter to enable this new code path (default off).